README.md

```diff
@@ -0,0 +1,6 @@
+# Development
+Developing new computers
+This Project is solely based off of the current situation the world is in. My Goal is to create a computer(super Computer) to create a new system, server, network, etc., to control the resources we have
+which should have been a freely open resource that can be obtain for free. This will also reduce the workload by pretty much everyone will be unemployed.
```

Comment:
We transfer all funds to the government, meaning that the government is going to start figuring out a budget for what it takes to take care of an individual & families.